### DISCUSSION TITLE
Where is the repo for the website?

### DISCUSSION THREAD

**Discussion by brettcannon** [extism/extism#114] [Q&A]
The example at https://extism.org/docs/integrate-into-your-codebase/python-host-sdk is missing a couple Pythonic tweaks, but I don't know where to submit a PR.

**Comment by zshipko** [comment]
Thanks, that would definitely be appreciated!

The process of updating the docs is a little tricky right now, we’re still working on making them public. In the meantime you could make PR to update the Python example (https://github.com/extism/extism/blob/main/python/example.py) in this repo and we can also update the docs on our end.

**Reply by brettcannon** [reply]
https://github.com/extism/extism/pull/242

**Reply by nilslice** [reply]
Thanks, @brettcannon - will review and merge asap!

**Comment by mattjohnsonpint** [comment]
Hi, I had the same question about the Go example, because the one in the docs doesn't compile.  Though after finding this, I checked the Go sample in the repo and it works just fine.  I think they are out of sync.

**Reply by nilslice** [reply]
Hi @mattjohnsonpint - we're in the process of updating the docs to point to the _new_ Go SDK:

https://github.com/extism/go-sdk

This doesn't require the shared library and is in pure Go. We'd recommend using this one over the one you've found the example for. 

There is an example using the new SDK in this repo: 

https://github.com/extism/extism-fsnotify

**Reply by mattjohnsonpint** [reply]
Thanks, I'll give it a try! 👍

**Reply by mattjohnsonpint** [reply]
Love the new SDK.  Much simpler usage than the old one.  Thanks!

**Comment by mattjohnsonpint** [comment]
Also, the docs should mention that `LD_LIBRARY_PATH=/usr/local/lib` needs to be set or the example will fail.

**Reply by nilslice** [reply]
Thank you! The new Go SDK (mentioned & linked in the comment above) solves this by removing the use of the shared library altogether 🙏
